packages/codegen-doc/src/fragment-visitor.ts

```diff
@@ -91,14 +91,6 @@ export class FragmentVisitor {
         `fragment ${node.name} on ${node.name} {
           __typename
           ${printLines(node.fields.sort())}
-          ${(
-            this._context.interfaceImplementations[node.name]?.map(
-              obj => `
-                ... on ${obj.name.value} {
-                  ... ${obj.name.value}
-                }`
-            ) ?? []
-          ).join("\n")}
```

Comment:
Why remove this? Not needed?

Reply:
It's more or less a work around this [issue](https://github.com/apollographql/federation/issues/1257 ) for which there is no easy solution yet. 

This code expands interface fields into inline fragments. Which, because of different return types (`Project!` vs `Project` and `Team!` vs `Team` in interface implementations of `NotificationSubscription`) result in a GraphQL validation error. It seems like we don't need these inline fragments with our current schema as the plugin already generates a standalone document for each of these fragment. 

This is something we might need to revisit in the future.